webui/react/src/hooks/useModal/Checkpoint/useCheckpointFlow.tsx

```diff
@@ -22,12 +23,14 @@ export const useCheckpointFlow = ({
   config: ExperimentConfig;
   title: string;
 }): Return => {
+  const modelCreateModal = useModal(ModelCreateModal);
   const {
     contextHolder: modalCheckpointRegisterContextHolder,
     modalOpen: openModalCheckpointRegister,
   } = useModalCheckpointRegister({
     onClose: (reason?: ModalCloseReason, checkpoints?: string[]) => {
-      if (checkpoints) openModalCreateModel({ checkpoints });
+      // TODO: fix the behavior along with checkpoint modal migration
```

Comment:
same comment as on useCheckpointFlow re: what needs to be fixed here in the future